src/components/FilterAttachedResults/FilterAttachedResults.ts

```diff
@@ -0,0 +1,149 @@
+import {
+    Component,
+    ComponentOptions,
+    IComponentBindings,
+    $$,
+    Initialization,
+    Checkbox,
+    QueryEvents,
+    IBuildingQueryEventArgs,
+    QueryStateModel,
+    load,
+    IAttributesChangedEventArg
+} from 'coveo-search-ui';
+
+export interface IAnalyticsAttachedResultsFilterMeta {
```

Comment:
Metadata?

Reply:
its the standard to suffix only with Meta !